### PR TITLE
PREDICTS database incorporated in reference to

### PR DESCRIPTION
**Pull request by henrykironde**

### CODE REVIEW

**Comment by henrykironde** [comment]
PR ref : #821

**Comment by henrykironde** [comment]
Looks like Block(column 17)  is described as an integer in the [data description](http://data.nhm.ac.uk/dataset/the-2016-release-of-the-predicts-database/resource/78dac1a9-6aa0-4dcb-9750-50df04f8ca6e), however that is wrong.

**Comment by henrykironde** [comment]
Looks ready for review.

File: scripts/predicts.py [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+        self.keywords = ['biodiversity', 'anthropogenic pressures']
+        self.retriever_minimum_version = "2.0.dev"
+        self.version = "1.0.0"
+        self.description = "A dataset of 3,250,404 measurements, collated from 26,114 sampling locations in 94 " \
+                           "countries and representing 47,044 species."
+
+        if parse_version(VERSION) <= parse_version("2.0.0"):
+            self.shortname = self.name
+            self.name = self.title
+
```
Comment (ethanwhite) [inline scripts/predicts.py]: Needs `self.tags = self.keywords` in the conditional for 2.0 compatibility.

Comment (henrykironde) [inline scripts/predicts.py]: @ethanwhite done

**Pull request merged**